packages/common/src/eips/4788.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "EIP-4788",
+  "number": 4788,
+  "comment": "Beacon block root in the EVM",
+  "url": "https://eips.ethereum.org/EIPS/eip-4788",
+  "status": "Draft",
+  "minimumHardfork": "cancun",
```

Comment:
I know it's somewhat counter intuitive and I wonder if we ever "get this in" reliably 😋, but the minimum HF here is not the inclusion HF but St least one lower (do here Shanghai would be a good choice), otherwise this can't be used in a Shanghai+EIP way.

So this is to read: the state of the network where all preconditions are fulfilled so that this EIP can be activated.

Reply:
So just to re-iterate on this and make this very concrete: this is the situation which breaks otherwise (and currently does) if the minimum HF is set to Cancun here:

```shell
➜  vm git:(master) ts-node
> import { Chain, Common, Hardfork } from '@ethereumjs/common'
undefined
> let c = new Common({ chain: Chain.Mainnet, eips: [ 4788 ] })
/ethereumjs-monorepo/packages/common/dist/cjs/common.js:361
                throw new Error(`${eip} cannot be activated on hardfork ${this.hardfork()}, minimumHardfork: ${minHF}`);
                ^

Uncaught Error: 4788 cannot be activated on hardfork shanghai, minimumHardfork: false
    at Common.setEIPs (/ethereumjs-monorepo/packages/common/src/common.ts:455:15)
    at new Common (/ethereumjs-monorepo/packages/common/src/common.ts:241:12)
    at /ethereumjs-monorepo/packages/vm/<repl>.ts:2:9
    at Script.runInThisContext (node:vm:129:12)
    at runInContext (/node/v18.14.2/lib/node_modules/ts-node/src/repl.ts:673:19)
    at Object.execCommand (/node/v18.14.2/lib/node_modules/ts-node/src/repl.ts:639:28)
    at /node/v18.14.2/lib/node_modules/ts-node/src/repl.ts:661:47
    at Array.reduce (<anonymous>)
    at appendCompileAndEvalInput (/node/v18.14.2/lib/node_modules/ts-node/src/repl.ts:661:23)
    at evalCodeInternal (/node/v18.14.2/lib/node_modules/ts-node/src/repl.ts:222:12)
>
```

Reply:
Not sure, since this is such a re-occuring place for stumbling over it might be really worth to rename to something like the following? 🤔

`minimumHardforkForIsolatedEIP`

Even if the name is a bit bulky.